### PR TITLE
[Merged by Bors] - fix(frontends/lean/definition_cmds): export `using_well_founded` AST node

### PR DESCRIPTION
**Pull request by digama0**
Turns out that the ast.json export has been broken this whole time in ignoring definitions which use `using_well_founded`. cc: @gebner , can we get a point release out with this bugfix?

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,12 @@
 {
     "cmake.sourceDirectory": "${workspaceFolder}/src",
+    "lean.executablePath": "bin/lean",
+    "files.associations": {
+      "*.lean": "lean",
+      "*.hl": "isabelle-ml",
+      "*.sdr": "glsl",
```
Comment (gebner) [inline .vscode/settings.json:7]: Can you please revert this change?

**Review by gebner — COMMENTED** [review]
bors d+

File: .vscode/settings.json (hunk 1 above)
Comment (digama0) [reply .vscode/settings.json]: oops, not sure how that got in there. --assume-unchanged is always finicky...
